### PR TITLE
UI Fixes / Debt

### PR DESCRIPTION
**Pull request by francisschmaltz**
Fixes: https://github.com/sourcegraph/sourcegraph/issues/2611
Fixes: https://github.com/sourcegraph/sourcegraph/issues/2031
Fixes: #2654

Also Fixes: Mobile issue where sticky header would appear behind results

New SSO Sign out page UI:
screen shot 2019-03-06 at 1 55 34 pm

### CODE REVIEW

**Comment by francisschmaltz** [comment]
@beyang can you give the sign out templating a once over

File: web/src/search/results/SearchResultsInfoBar.scss [hunk 1]
```diff
@@ -6,6 +6,10 @@
     padding-bottom: 0.25rem;
     padding-top: 1rem;
 
+    .alert {
```
[the remark this answers is not in the document]
Comment (francisschmaltz) [reply web/src/search/results/SearchResultsInfoBar.scss]: It's meant to affect any/every alert that would appear in the search bar. I think it's simpler to keep it as the standard alert element. Is this okay?

**Comment by francisschmaltz** [comment]
Note on the UI of the signed out page: This should be a placeholder until a more fully designed page and be mocked and designed. @francisschmaltz

File: cmd/frontend/internal/app/sign_out.go [hunk 2]
```diff
@@ -63,14 +63,16 @@ func renderSignoutPageTemplate(w http.ResponseWriter, r *http.Request, signoutUR
...
-<br>
+<style>.auth-box,.center{position:relative}.auth-box,a,body{display:block}a,body,h1{text-align:center}html{-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%}body{background-color:#0e121b}*{font-family:system,-apple-system,San Francisco,\.SFNSDisplay-Regular,Segoe UI,Segoe,Segoe WP,Helvetica Neue,helvetica,Lucida Grande,arial,sans-serif;font-size:14px;color:#f2f4f8}.auth-box{box-sizing:border-box;width:100%;max-width:420px;margin:10vh auto auto;overflow:hidden;border:1px solid #2B3750;background-color:#1D2535;box-shadow:0 0 8px 0 rgba(0,0,0,.4);padding:14px;border-radius:6px}@media screen and (max-width:678px){.auth-box{border:none;margin-top:64px;background-color:#0e121b}}.logo{margin-top:28px;width:atuo}.center{padding-top:48px}h1{font-size:22px;font-weight:500;margin-top:40px}a{font-size:14px;line-height:20px;border:none;border-radius:2px;background-color:#2b3750;margin-bottom:20px;text-decoration:none;word-wrap:normal;transition:opacity 250ms ease;padding:6px 12px}a:last-of-type{margin-bottom:0}a:hover{opacity:.85}a:active,a:target{opacity:.7}.primary{margin-top:14px;color:#fff;background-color:#1663a9;border-color:#155d9e;margin-bottom:48px}</style>
```
Comment (beyang) [inline cmd/frontend/internal/app/sign_out.go]: Can this be unminified? Also, what is the source of these colors? If these are derived from the colors/styles we use in the single-page app, can that be documented here / how would we recover/regenerate these if we changed styling on the frontend?

Comment (francisschmaltz) [reply cmd/frontend/internal/app/sign_out.go]: Sure! And I can list the colors I used or add links to the colors in zeplin

Comment (beyang) [reply cmd/frontend/internal/app/sign_out.go]: Cool, doesn't have to be super automatic; a comment describing how to regenerate these would suffice.

**Pull request merged**